### ISSUE TITLE
Disable json response formatting for large response?

### ISSUE THREAD

**Issue by arisliang** [domaindrivendev/Swashbuckle.AspNetCore#1980]
The response section hangs if the return data is big, it seems that json coloring and formatting takes most of the time.

Is it possible to implement some logic to disable the coloring and formatting if the response is too big? No color and format is better than ui hangs, which become unusable.

v5.6.3

**Comment by domaindrivendev** [comment]
This sounds is more of a question for the [swagger-ui folks](https://github.com/swagger-api/swagger-ui) rather than Swashbuckle, and you should be posting there instead of SB.

With that said, I took a quick look for you and can see that they do indeed have a config option to disable `syntaxHighlighting`:
See https://github.com/swagger-api/swagger-ui/blob/master/docs/usage/configuration.md

To use this via Swashbuckle, the following should work:

```csharp
c.UseSwaggerUI(c =>
{
    ...
    c.ConfigObject.AdditionalItems["syntaxHighlight"] = false;
});
```

**Comment by domaindrivendev** [comment]
UPDATE: It looks like this feature isn't supported in the version of swagger-ui that comes with SB `5.6.3`. However, a newer version that _does_ support it will be shipped with SB `6.0.0`, which I'm planning to release this week.
